end_to_end/3-mitigate-bias-train-model2-registry-e2e.ipynb

```diff
@@ -0,0 +1,1340 @@
+{
```

Comment:
<p>remove all output that shows ARN</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>done</p>